catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/geofeature/FeatureService.java

```diff
@@ -0,0 +1,15 @@
+package org.codice.ddf.catalog.ui.query.geofeature;
+
+import java.util.List;
+
+/**
+ * <p>
+ * <b> This code is experimental. While this interface is functional and tested, it may change or be
+ * removed in a future version of the library. </b>
+ * </p>
+ */
+
+public interface FeatureService {
+    List<String> getSuggestedFeatureNames(String query, int maxResults);
```

Comment:
While it may be obvious what these methods are doing, it would be nice to have javadoc on this interface